shared/src/main/java/uk/gov/di/authentication/shared/services/ConfigurationService.java

```diff
@@ -655,15 +655,7 @@ public URI getMfaResetCallbackURI() {
     }
 
     public String getIPVAuthEncryptionPublicKey() { //
-        var paramName = format("{0}-ipv-public-encryption-key", getEnvironment());
-        try {
-            var request =
-                    GetParameterRequest.builder().withDecryption(true).name(paramName).build();
-            return getSsmClient().getParameter(request).parameter().value();
-        } catch (ParameterNotFoundException e) {
-            LOG.error("No parameter exists with name: {}", paramName);
-            throw new RuntimeException(e);
-        }
+        return System.getenv().getOrDefault("IPV_AUTHORIZATION_PUBLIC_KEY", "");
```

Comment:
Where do we envision the error handling for this being not set i.e. defaulting to empty string? E.g. we had handling of the `ParameterNotFoundException` here before.